neurom/core/types.py

```diff
@@ -28,11 +28,18 @@
 
 '''Type enumerations'''
 
-from enum import unique
+from enum import unique, Enum
 
 from neurom.utils import OrderedEnum
 
 
+@unique
+class NeuriteIter(Enum):
+    '''Neurite iteration orders'''
+    Natural = 1  # Order in which neurites appear in the file
```

Comment:
Natural is too ambiguous, imo, 'FileOrder' I think makes more sense.

Reply:
Good point